app_secret.go

```diff
@@ -15,10 +15,12 @@ const (
 )
 
 func newAppId(appSecret []byte) []byte {
-	publicKey := appSecret[AppSecretSize-AppPublicKeySize : AppSecretSize]
-	author := make([]byte, 32)
-	payload := append([]byte{appCreationNature}, author...)
-	payload = append(payload, publicKey...)
+	pKeyStart, pKeyEnd := AppSecretSize-AppPublicKeySize, AppSecretSize
```

Comment:
`p` would not really be helpful in this context, because is it `p = public` or `p = private` ?

`pubKeyStart` and `pubKeyEnd` would work well though.

Reply:
Right, did not have that in mind!